src/org/usfirst/frc2175/subsystem/intake/DreamIntakeSubsystem.java

```diff
@@ -17,6 +20,8 @@ public DreamIntakeSubsystem(RobotConfig robotConfig) {
         dreamIntakeSideBeltTalon = wiringConfig.getDreamIntakeSideBeltTalon();
         dreamIntakeMainBeltTalon = wiringConfig.getDreamIntakeMainBeltTalon();
         dreamIntakeLiftTalon = wiringConfig.getDreamIntakeLiftTalon();
+        this.dreamIntakeUpSwitch = wiringConfig.getDreamIntakeUpSwitch();
+        this.dreamIntakeDownSwitch = wiringConfig.getDreamIntakeDownSwitch();
```

Comment:
For consistency's sake, let's switch all of them to have `this.` before the variable name.